cypress/integration/dib-travel/my-travels/my-travels.spec.ts

```diff
@@ -0,0 +1,128 @@
+describe('My travels', () => {
+  beforeEach(() => {
+    cy.login();
+
+    cy.get('dib-navbar .home-page').contains(' My Travels ').click();
+  });
+
+  it('should display "My Travels" in the navbar', () => {
+    cy.get('dib-navbar .navbar__content').contains(' add trip ').should('exist');
+  });
+
+  it('should displays message that there are no active travels', () => {
+    cy.get('app-my-travels dib-travels-list .table-container.empty-list .empty-trips-section').should(
+      'contain',
+      " You don't have any active travels, but you can always create one "
+    );
+  });
+
+  it('should displays button to create new travel', () => {
+    cy.get('app-my-travels dib-travels-list .table-container.empty-list .dib-button-new').should(
+      'contain',
+      'Create new travel '
+    );
+  });
+
+  it('should check search for past travels', () => {
+    cy.get('app-my-travels .mat-tab-links').contains(' Past ').click();
+
+    cy.get('app-my-travels dib-travels-list dib-input input').type('CYQA');
+
+    cy.get('app-my-travels dib-travels-list .name').should('have.text', 'CYQA Bot bussines trip');
```

Comment:
You can put 'CYQA Bot business trip' data in a JSON file and call it from there, you shouldn't hard coded this. Also,  you wrote incorrect "business" here :)

Reply:
The name of the trip was incorrect. So I'll change that. 

Because I've used that date just once, I didn't put it in the JSON file. 

@sjova can you please correct me if I'm am wrong?